src/test/functional/tests/buzzPage.spec.ts

```diff
@@ -16,12 +19,21 @@ test.describe("Buzz Page", () => {
     await loginPage.loginUser(adminUserTestData.userName, adminUserTestData.password);
   });
 
-  test("Test Case 1: Share file and Delete file", async ({ page }) => {
+  test("Test 1: Post Should Be Shared and Deleted", async ({ page }) => {
     await buzzPage.navigateToSubPage(buzzPage.buzzPageButton);
-    await buzzPage.sharePhotos();
+    await buzzPage.sharePhotos(filePath);
     await page.reload();
     await expect(buzzPage.photoImg.first()).toBeAttached();
     await buzzPage.deletePhotos();
     await expect(buzzPage.noPostParagraph.first()).toHaveText(buzzPageAssertions.noPostAvailable);
```

Comment:
Sorry, but this test is completely useless. Watermelon test.
I've run it against online version of a page. It's green.
So I've commented line 27 `await buzzPage.deletePhotos();` - still green
Additionally commented #24 `await buzzPage.sharePhotos(filePath);` - still green... :(